libs/dextool/source/dextool/compilation_db/package.d

```diff
@@ -714,16 +730,25 @@ ParseFlags parseFlag(CompileCommand cmd, const CompileCommandFilter flag_filter,
             skip,
             /// skip the next arg, if it is not a flag
             skipIfNotFlag,
+            /// use the next arg to create a complete path
+            checkingForQuotation,
         }
 
         import std.path : buildNormalizedPath, absolutePath;
-        import std.array : appender, array;
+        import std.array : Appender, appender, array, join;
         import std.range : ElementType;
 
         auto st = State.keep;
         auto rval = appender!(string[]);
         auto includes = appender!(string[]);
         auto compiler = Compiler(r.length == 0 ? null : r[0]);
+        Appender!(string[]) path;
```

Comment:
The type should probably be `string` because the purpose is to go through an array of strings (`string[]`) to create the sum of them as a path.